tests/bulk_create/tests.py

```diff
@@ -51,6 +54,30 @@ def test_inheritance(self):
             "Nicholas's",
         ], attrgetter("name"))
 
+    def test_proxy_inheritance(self):
+        """Ticket 24997: Support bulk_create on proxy models."""
+        ProxyCountry.objects.bulk_create([
+            ProxyCountry(name="Qwghlm", iso_two_letter="QW"),
+            Country(name="Tortall", iso_two_letter="TA"),
+        ])
+        self.assertQuerysetEqual(ProxyCountry.objects.all(), {
+            "Qwghlm", "Tortall"
+        }, attrgetter("name"), ordered=False)
+        ProxyProxyCountry.objects.bulk_create([
+            ProxyProxyCountry(name="Neitherlands", iso_two_letter="NT")
+        ])
+        self.assertQuerysetEqual(ProxyProxyCountry.objects.all(), {
+            "Qwghlm", "Tortall", "Neitherlands",
+        }, attrgetter("name"), ordered=False)
+        with self.assertRaises(ValueError):
```

Comment:
Other tests in this module use `assertRaises` instead of `assertRaisesMessage`. I agree that `assertRaisesMessage` (or `assertRaisesRegex`) is (are) better. Should I change every instance of `assertRaises` that tests this particular error, or only those instances of it added in this pull request?